ci/test-rhoscala-rbls.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+source ci/header.sh
+
+for rbl_file in $(ls ${rholang_root_dir}/tests/*.rbl); do
+    out=$([[ $( ulimit -s unlimited && export ESS_SYSDIR=${rosette_root_dir}/rbl/rosette && ${rosette_root_dir}/build.out/src/rosette --quiet --boot-dir=${rosette_root_dir}/rbl/rosette --boot=boot.rbl ${rbl_file} | grep ^Pass ) ]] && echo true || echo false)
```

Comment:
Line too long and you're doing to much. Please rewrite using variables, so that the lines are shorter.

Reply:
Also, if I believe that `--boot=boot.rbl` is the default. Try running rosette without that option. I think it will still work correctly.